src/DataStructures/CachedTempBuffer.hpp

```diff
@@ -61,20 +60,19 @@ class CachedTempBuffer {
   TempBuffer<tmpl::list<Tags...>> data_;
   tuples::TaggedTuple<Computed<Tags>...> computed_flags_{
       ((void)Tags{}, false)...};
-  Computer computer_;
 };
 
 namespace CachedTempBuffer_detail {
-template <typename Computer, typename Tags>
+template <typename Tags>
 struct CachedTempBufferFromTypelistImpl;
-template <typename Computer, typename... Tags>
-struct CachedTempBufferFromTypelistImpl<Computer, tmpl::list<Tags...>> {
-  using type = CachedTempBuffer<Computer, Tags...>;
+template <typename... Tags>
+struct CachedTempBufferFromTypelistImpl<tmpl::list<Tags...>> {
+  using type = CachedTempBuffer<Tags...>;
 };
 }  // namespace CachedTempBuffer_detail
 
 /// Instantiate a `CachedTempBuffer` from a typelist instead of a parameter pack
-template <typename Computer, typename Tags>
+template <typename Tags>
 using cached_temp_buffer_from_typelist =
```

Comment:
This is my fault, but since you're touching it, the entire implementation of this metafunction, including the detail namespace above, can be replaced by
```c++
template <typename Tags>
using cached_temp_buffer_from_typelist = tmpl::wrap<Tags, CachedTempBuffer>;
```

Reply:
Done, and squashed this in directly. I believe the fixup fixed the ASAN issue.